### ISSUE TITLE
[SM] Skills doing lower dmg than tooltip

### ISSUE THREAD

**Issue by CrimsonFenris** [WarEmu/WarBugs#18788]
**Expected behavior and actual behavior:**
Some Swordmaster skills are actually hitting for less damage than tooltip, even with meaningful Melee Power (wich is never accounted for tooltip values, usually resulting to **more** final damage).

Either tooltip value is too high or the actual damage is too low, need a bit of checking... ;)

### Affected skills are :
- **Crashing Wave** (kd mono, Hoeth Tree)
- **Shock** (Spirit add proc from Blurring Shock core skill)

**Steps to reproduce the problem:**
Tested with 406 Strength + 96 Melee Power, and 506 Strength + 96 Melee Power, 
For **Crashing Wave** : 
- 526 (574 in tooltip) => 10% less than expected on tooltip (at least, since Melee Power should bring more dmg than tooltip)
- 584 (634 in tooltip)

For **Shock** : 
- 106 (108 in tooltip) => It's not a scaled skill on Strength or any other damage increaser, but the 2 points difference means it's not a rounding down issue.

**Testing Screenshots/Videos/Evidences (always needed):**

Crashing Wave with 406 + 96 MP : 
![SM_Bug_Crashing_Wave_01](https://user-images.githubusercontent.com/40425011/113724190-65da7600-96f2-11eb-81ff-fdd24190f4ad.jpg)

Crashing Wave with 506 + 96 MP : 
![SM_Bug_Crashing_Wave](https://user-images.githubusercontent.com/40425011/113724279-7be83680-96f2-11eb-993c-ab947a657e6f.jpg)

Shock Proc
![SM_Bug_Shock](https://user-images.githubusercontent.com/40425011/113724323-873b6200-96f2-11eb-9e2d-67bd1d23bc95.jpg)

**Comment by lilimx** [comment]
Hey,

thank you for your ticket.
Please bare in mind, that the training constructors also have resistence and other def stuff compared to the simple dummies.
![grafik](https://user-images.githubusercontent.com/23266079/122668797-22a08700-d1ba-11eb-9ad8-a71fa8f3f2e5.png)


If your the opinion the dmg is still not correct compared to the tooltip, please redo your test.

**Comment by CrimsonFenris** [comment]
Yes, I'm talking about the total damage shown in the log (including mitigated portion under parenthesis). Total damage on target dummies is exactly the same (with zero mitigation of course).

I tested several skills and all are doing the exact tooltip damage (rounded up or down sometimes like for Shock procs), but crashing wave shows a far greater delta that is clearly not a rounded number, as mentioned.

I came to the conclusion tooltip is erroneous for CW damage (in either way).
